subprojects/gradle/test-project/src/main/kotlin/com/avito/test/gradle/TestProperty.kt

```diff
@@ -0,0 +1,19 @@
+package com.avito.test.gradle
+
+/**
+ * property passed through gradle (Test task).systemProperty
+ * see build.gradle.kts
+ */
+fun getTestProperty(name: String, defaultValue: String? = null): String {
```

Comment:
```suggestion
fun getSystemProperty(name: String, defaultValue: String? = null): String {
```
I see nothing about tests in implementation

Reply:
fixed; also moved to kotlin-dsl-support